src/nisarqa/validate/sanity_checks.py

```diff
@@ -435,6 +435,9 @@ def _verify_data_is_in_list(
             "plannedObservationId",
         ]
 
+    if product_type.upper().startswith("G"):
```

Comment:
Haha we need some helper functions like

```python
def is_geocoded_product(product_type: str) -> bool:
    ...
```

and

```python
def is_insar_product(product_type: str) -> bool:
    ...
```

Reply:
Oh sorry, it probably wasn't clear -- please don't implement these helper functions in this PR :pray:. It's good-to-go as is.

Reply:
Already raised as an Issue #71 ;)